default/mail_tt2/report.tt2

```diff
@@ -487,7 +487,7 @@
   [%|loc%]List name has been lowercased[%END%]
 
 [%~ ELSIF report_entry == 'auto_aliases' ~%]
-  [%|loc%]Aliases have been installed.[%END%]
+  [%|loc%]Aliases have been installed.[%END%] ([% report_param.list_name %])
```

Comment:
`[% list.name %]` and `[% domain %]` seem to be defined.  cf. [Sympa::Message::Template](https://github.com/sympa-community/sympa/blob/b17e3bb89b904d90704978afe3b9e24bb32f4cf1/src/lib/Sympa/Message/Template.pm#L142-L173)

Reply:
` [% domain %] ` work when using the web interface, not with `sympa.pl --open_list=list@example.org`, and `[% list.name %]` don’t work at all.

Reply:
Ok, I'll investigate it in the next week (fyi, I found usage of `[% listname %]` but no `[%list_name%]`).

Reply:
Do you prefer `listname` over `list_name`? I have no pb to change it.

Reply:
Changed to `listname`